flocker/node/agents/testtools/_cinder.py

```diff
@@ -3,14 +3,19 @@
 """
 Test helpers for ``flocker.node.agents.cinder``.
 """
-
 from zope.interface.verify import verifyObject
 
 from flocker.testtools import TestCase
 from ..cinder import (
     ICinderVolumeManager,
     INovaVolumeManager,
 )
+from ._blockdevice import require_backend
+
+
+# Raise SkipTest unless the functional test configuration has an openstack
+# backend.
+require_backend_openstack = require_backend('openstack')
```

Comment:
Any particular reason to make this different than the other require backends? (i.e. just use require_backend('openstack') everywhere?)

Not worth changing / both are obviously equally expressive. Just curious if there is an underlying reason.